triggers/service/src/main/scala/com/digitalasset/daml/lf/engine/trigger/dao/DbTriggerDao.scala

```diff
@@ -94,23 +94,19 @@ final class DbTriggerDao private (dataSource: DataSource with Closeable, xa: Con
 
   private def insertRunningTrigger(t: RunningTrigger): ConnectionIO[Unit] = {
     val insert: Fragment = sql"""
-        insert into running_triggers values (${t.triggerInstance}, ${t.triggerParty}, ${t.triggerName})
+        insert into running_triggers values (${t.triggerInstance}, ${t.triggerParty}, ${t.triggerName}, ${t.triggerToken})
       """
     insert.update.run.void
   }
 
   private def queryRunningTrigger(triggerInstance: UUID): ConnectionIO[Option[RunningTrigger]] = {
     val select: Fragment = sql"""
-        select (trigger_instance, trigger_name, trigger_party) from running_triggers
+        select trigger_instance, full_trigger_name, trigger_party, access_token from running_triggers
         where trigger_instance = $triggerInstance
       """
     select
-      .query[(UUID, Identifier, Party)]
-      .map {
-        case (instance, name, party) =>
-          // TODO[AH] use query.to[RunningTrigger] once the token is persisted.
-          RunningTrigger(instance, name, party, None)
-      }
+      .query[(UUID, Identifier, Party, Option[String])]
```

Comment:
Maybe worth newtyping the token?

Reply:
Good idea, I'll do that in a follow up since it'll touch a couple places out of scope of this PR.

Reply:
Tracked in https://github.com/digital-asset/daml/issues/7901